reVX/setbacks/solar_setbacks.py

```diff
@@ -0,0 +1,219 @@
+# -*- coding: utf-8 -*-
+"""
+Compute setbacks exclusions
+"""
+import logging
+import os
+import geopandas as gpd
+
+from rex.utilities import log_mem
+
+from reVX.setbacks.base import BaseSetbacks
+
+
+logger = logging.getLogger(__name__)
+
+
+class ParcelSetbacks(BaseSetbacks):
+    """
+    Parcel setbacks, using negative buffers.
+    """
+
+    def compute_generic_setbacks(self, features_fpath):
+        """Compute generic setbacks.
+
+        Parameters
+        ----------
+        features_fpath : str
+            Path to shape file with features to compute setbacks from.
+
+        Returns
+        -------
+        setbacks : ndarray
+            Raster array of setbacks
+        """
+        logger.info('Computing generic setbacks')
+        setback_features = self._parse_features(features_fpath)
+
+        setbacks = [
+            (geom, 1) for geom in setback_features.buffer(0).difference(
+                setback_features.buffer(-1 * self.generic_setback)
+            )
+        ]
+
+        return self._rasterize_setbacks(setbacks)
+
+    @staticmethod
+    def _compute_local_setbacks(features, cnty, setback):
+        """Compute local features setbacks.
```

Comment:
For all the "local" and "generic" setback methods, can we clarify that local is with a county-specific regulations file and generic is based on the base height * multiplier?

Reply:
Yup, done!